src/styles/globalStyles.js

```diff
@@ -0,0 +1,16 @@
+import { createGlobalStyle } from "styled-components";
+
+export const GlobalStyle = createGlobalStyle`
+  *,
+  *::before,
+  *::after {
+    box-sizing: border-box; 
+  }
+  
+  body {
+    margin: 0;
+    padding: 0; 
+    font-size: 62.5%;
+    color: ${({ theme }) => theme.colors.secondaryDark};
+  }
+`;
```

Comment:
we will need to move some more styles from styles.css to GlobalStyles I added in the meantime to it (background).